pkg/cmd/run.go

```diff
@@ -108,6 +108,7 @@ func newCmdRun(rootCmdOptions *RootCmdOptions) (*cobra.Command, *runCmdOptions)
 	cmd.Flags().Bool("sync", false, "Synchronize the local source file with the cluster, republishing at each change")
 	cmd.Flags().Bool("dev", false, "Enable Dev mode (equivalent to \"-w --logs --sync\")")
 	cmd.Flags().Bool("use-flows", true, "Write yaml sources as Flow objects in the integration custom resource")
+	cmd.Flags().String("operator-id", "camel-k", "Operator id selected to manage this integration.")
```

Comment:
I'd add a "x" short param as well, so we can use "-x" instead the longer "--operator-id".

Reply:
not sure what the best shortcut would look like. `-o` is already taken `-id` will be a bit misleading. Any ideas?

Reply:
Literally `-x` :)